tests/std/tests/LWG2597_complex_branch_cut/test.cpp

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <cmath>
+#include <complex>
+#include <limits>
+
+using namespace std;
+
+template <typename T>
+void check(T x, T y) {
+    constexpr T tol = 2 * numeric_limits<T>::epsilon();

```

Comment:
Suggestion: `tol` is highly abbreviated, and the meaning might not be clear to all readers. I think `tolerance` would be more readable, without being overly verbose. Occurs below.